src/app/@themes/styles/global/_auth.scss

```diff
@@ -6,10 +6,24 @@
   line-height: 2.688rem;
   color: $dark-100;
   margin-bottom: 0.875rem;
+  width: 26rem;
```

Comment:
why do u repeat this many times?
apply this width to the container and add for others max-width: 100%